content/docs/rfcs/13/README.md

```diff
@@ -122,7 +124,11 @@ RPC call to query historical messages.
   Leaving this field empty means no filter on the content topic of message history is required.
   This field SHOULD be left empty in order to retrieve historical waku messages regardless of their content topics.
 - `PagingInfo` holds the information required for pagination.  Its `pageSize` field indicates the number of  `WakuMessage`s to be included in the corresponding `HistoryResponse`. If the `pageSize` is zero then no pagination is required. If the `pageSize` exceeds a threshold then the threshold value shall be used instead. In the forward pagination request, the `messages` field of the `HistoryResponse` shall contain at maximum the `pageSize` amount of waku messages whose `Index` values are larger than the given `cursor` (and vise versa for the backward pagination). Note that the `cursor` of a `HistoryQuery` may be empty (e.g., for the initial query), as such, and depending on whether the  `direction` is `BACKWARD` or `FORWARD`  the last or the first `pageSize` waku messages shall be returned, respectively.
-The queried node MUST sort the `WakuMessage`s based on their `Index`, where the `receivedTime` constitutes the most significant part and the `digest` comes next, and then perform pagination on the sorted result. As such, the retrieved page contains an ordered list of `WakuMessage`s from the oldest message to the most recent one.
+The queried node MUST sort the `WakuMessage`s based on their `Index`, where the `senderTime` constitutes the most significant part and the `digest` comes next, and then perform pagination on the sorted result. As such, the retrieved page contains an ordered list of `WakuMessage`s from the oldest message to the most recent one.
+
+Alternatively, the `receiverTime` (instead of `senderTime` ) can be used to sort `WakuMessage`s during the paging process. 
+However, we RECOMMEND the use of the `senderTime` for sorting as it is invariant and consistent across all the nodes.
```

Comment:
maybe mention trade-off?

Also, this means we are putting a timing requirement on nodes? Should we mention this somewhere too? https://github.com/vacp2p/rfc/issues/219#issuecomment-714240812

Reply:
security consideration can be in separate section, and here we can mention future work as blockheight eg mentioned here https://github.com/vacp2p/rfc/discussions/397

Reply:
>maybe mention trade-off? i.e. security risk (and possible mitigation, for example what core does)

Thanks for the comment @oskarth 
I have mentioned the reusability of the cursor across different store nodes when the sender time is utilized as opposed to the receiver time.

Reply:
At the moment, there is no timing requirement on the nodes for the sake of pagination, as we do not attempt to provide consistent paging service across all the store nodes. 
The following comment https://github.com/vacp2p/rfc/issues/219#issuecomment-714240812 is about querying from multiple store nodes which as I said  is a different scenario and we can achieve in the "store nodes state synchronization" protocol.
Please let me know if that makes sense.

Reply:
> security consideration can be in separate section, and here we can mention future work as blockheight eg mentioned here #397

Done

Reply:
great, thanks!